devel/cutter-rizin/Portfile

```diff
@@ -6,9 +6,9 @@ PortGroup           qt6 1.0
 PortGroup           cmake 1.1
 PortGroup           app 1.1
 
-github.setup        rizinorg cutter 2.3.4 v
+github.setup        rizinorg cutter 2.4.0-rc1 v
```

Comment:
stable ports should not be updated to pre-releases.

Reply:
But this is the only version that supports rizin 0.8

Reply:
then you should provably hold off updating both of the ports.

Reply:
We have 2.4.0 stable now